website/content/api-docs/secret/pki.mdx

```diff
@@ -1198,6 +1199,86 @@ $ curl \
 }
 ```
 
+### Sign Intermediate with External Policy <EnterpriseAlert inline="true" />
+
+Similar to [sign intermediate](#sign-intermediate), this endpoint accepts a
+CSR and returns a certificate with appropriate values for acting as an
+intermediate CA via an external policy engine. Any parameters passed to this
+endpoint are passed verbatim to the CIEPS engine<EnterpriseAlert inline="true" />.
```

Comment:
As the first usage in a section, it would be nice to write out CIEPS (as in "... are passed verbatim to the Certificate Issuance External Policy Service (CIEPS) <EnterpriseAlert...").

Reply:
Done in https://github.com/hashicorp/vault/pull/22371/commits/99cc574f857cc4859eebb0515971915cb1180a24, I updated a few other sections that had the same issue.